docker/runner.Dockerfile

```diff
@@ -87,6 +87,8 @@ USER runner
 WORKDIR /home/runner
 SHELL [ "/bin/bash", "-c" ]
 
+RUN source /opt/rye/env
```

Comment:
Can this go into a separate PR?

Reply:
Okay, will wait both to be merged so I update only once, and not 2 times